### PR TITLE
Reduce testing time

### PR DESCRIPTION
**Pull request by six42**
Reduce testing time

-     by reducing timeouts to the minimum,
-     deleting test case SlimConnectionTimeout which is a duplicate of C_TimeoutAsNoServerAtGivenSocket
-     skipping performance test in the standard build as it is a long running test and no assertion on the performance

### CODE REVIEW

**Comment by six42** [comment]
Comparing this build in GitHub with the previous one the improvement is as expected: 10-20 sec = 10-20% reduced time

### Old version 5 May
https://github.com/unclebob/fitnesse/actions/runs/8960309727/job/24606661122
Total time 3m 30s
Build with Gradle 3m 10s
Acceptance Tests
327 Tests,	0 Failures	64.340 seconds.

### this PR on Jun 30
https://github.com/unclebob/fitnesse/actions/runs/9733320118/job/26860095078
Total time in 3m 9s
Build with Gradle 2m 51s
Acceptance Tests
326 Tests,	0 Failures	56.499 seconds.

File: FitNesseRoot/FitNesse/SuiteAcceptanceTests/SuiteSlimTests/TestSystemSlimSuite/ErrorSituationsDuringSlimServerStartup/C_TimeoutAsNoServerAtGivenSocket/content.txt [hunk 1]
```diff
@@ -5,17 +5,23 @@ To test this we run in debug mode and request to start the slim server ourselfs.
 Use the property slim.timeout to define how long the test system should wait for the slim client. The default is 10 seconds.
 In this test we set it to 1 second.
 
-!| script |
-| create page | TestPage | with content | !-
+# To run this test interactive don't forget to add at the end "&remote_debug" to the rest test command
+#!define TEST_SYSTEM {SLIM}
```
Comment (fhoeben) [inline FitNesseRoot/FitNesse/SuiteAcceptanceTests/SuiteSlimTests/TestSystemSlimSuite/ErrorSituationsDuringSlimServerStartup/C_TimeoutAsNoServerAtGivenSocket/content.txt:9]: Why did you add these (commented out) lines?

File: FitNesseRoot/FitNesse/SuiteAcceptanceTests/SuiteSlimTests/TestSystemSlimSuite/TimeoutInSlimTestSystem/content.txt [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+!define slim.flags {-s 1}
+#Will pass
+|fitnesse.fixtures.Sleep|20|
+
+#Will fail
+|fitnesse.fixtures.Sleep|90000|
 -! |
-|test results for page|TestPage|should contain|The instruction timed out after 2 seconds|
-|test results for page in debug mode|TestPage|should contain|The instruction timed out after 2 seconds|
+|test results for page in debug mode|TestPage|should contain|The instruction timed out after 1 seconds|
```
Comment (fhoeben) [inline FitNesseRoot/FitNesse/SuiteAcceptanceTests/SuiteSlimTests/TestSystemSlimSuite/TimeoutInSlimTestSystem/content.txt:14]: Shouldn't we also have an assertion that the first one does pass (like you state in the comment)?

**Review by fhoeben — COMMENTED** [review]
Nice improvements. I added some minor questions
